src/app/providers/accessibility/__test__/accessibility.service.spec.ts

```diff
@@ -10,6 +11,7 @@ describe('AccessibilityService', () => {
     TestBed.configureTestingModule({
       providers: [
         AccessibilityService,
+        MobileAccessibility,
```

Comment:
We should now have the ability to add proper unit testing of all methods in the `AccessibilityService` seeing as we can now hook into this